lib/Model/Listing/AbstractListing.php

```diff
@@ -214,11 +215,11 @@ public function getOrderKey()
 
     /**
      * @param string|array $orderKey
-     * @param bool $quote
+     * @param bool $backticks
      *
      * @return $this
      */
-    public function setOrderKey($orderKey, $quote = true)
+    public function setOrderKey($orderKey, $backticks = true)
```

Comment:
why the rename?

Reply:
because these are not quotes, you want to say that you're using backticks in your value

Reply:
no, you are quoting the key, that is the point here.

Reply:
as we discussed lets using "escape" now